src/config/settings.js

```diff
@@ -19,5 +19,8 @@ export default {
   },
   workspace: {
     type: 'mosaic',
+  },
+  workspaceControlPanel: {
+    enabled: true,
```

Comment:
I could easily be convinced to flatten this out.

Reply:
Are there other settings we would want to add here? I think it makes sense to have them grouped, as you have it already. There are a lot of open questions about the config situation. State needs to be normalised, but the config needs to be easily documented and human-memorable. Settings should exactly mirror the config (is how I'm thinking of it at the moment). From that point of view, I prefer what's already here.